backend/src/config/passport.ts

```diff
@@ -7,13 +7,17 @@ import { User, UserCreationAttributes } from "../models/user";
 const authStrategy = (passport: passport.PassportStatic): void => {
   const LocalStrategy = passportLocal.Strategy;
   // serialize
-  passport.serializeUser((serialUser: User, done) => {
-      done(null, serialUser.id);
+  passport.serializeUser((_user, done) => {
+      const user = _user as unknown as User
```

Comment:
passport uses Express.User, which is incompatible with our sequelize User, hence retyped to allow strict type checking to pass